src/iso15118/d20/state/authorization_setup.cpp

```diff
@@ -77,8 +88,7 @@ FsmSimpleState::HandleEventReturnType AuthorizationSetup::handle_event(Allocator
             return sa.PASS_ON;
         }
 
-        // Todo(sl): PnC is currently not supported
-        ctx.feedback.signal(session::feedback::Signal::REQUIRE_AUTH_EIM);
+        ctx.feedback.signal(required_auth_signal(res));
```

Comment:
We should move this feedback.signal call to the authorization state. Only in the authorization state is it clear whether the car is requesting PnC or EIM.